client/package.json

```diff
@@ -38,6 +39,7 @@
     "@types/react-dom": "^19.1.7",
     "@vitejs/plugin-react": "^5.0.0",
     "@vitest/coverage-istanbul": "^3.2.4",
+    "@vitest/coverage-v8": "^3.2.4",
```

Comment:
We already have the istanbul coverage reporter, we do not need v8, please remove it from the package.json file.